packages/botonic-plugin-contentful/src/cms/callback.ts

```diff
@@ -126,13 +140,11 @@ export class ContentId implements ValueObject {
         return new TopContentId(this.model, this.id).deliver(cms, context)
     }
   }
+}
 
-  toString(): string {
-    return `${this.model} with id '${this.id}'`
-  }
-
-  equals(other: ContentId): boolean {
-    return this.model === other.model && this.id === other.id
+export class AssetId extends ResourceId {
+  constructor(id: string, readonly assetType: string | undefined) {
```

Comment:
Why `AssetId` has the parameters in a different order (reversed order) than `ResourceId` and `ContentId`?
If it's because `assetType` can be `undefined`, why not define it this way?
`constructor(id: string, readonly assetType?: string) {`

If the `assetType` parameter must be declared in the constructor (even if it's undefined), I would change the order of the parameters to match the other classes.

Reply:
because in asset we have 2 type: the resourceType is asset and the assetType is eg. image